mindjunkies/courses/views.py

```
@@ -6,7 +6,9 @@
 from django.shortcuts import get_object_or_404, redirect, render
 from django.urls import reverse, reverse_lazy
 from django.views.decorators.http import require_http_methods
-from django.views.generic.edit import CreateView, UpdateView
+from django.views.generic.edit import CreateView, FormView, UpdateView
```

Ruff (0.8.2)

`django.views.generic.edit.FormView` imported but unused

Remove unused import: `django.views.generic.edit.FormView`

(F401)